.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,20 @@
+## `erGW` version
+[Put release version here ...](https://github.com/travelping/ergw/releases/tag/Release version here ...)
```

Comment:
https://github.com/travelping/eradius/releases/tag/Release is 404

Reply:
this is example - correct link will be eg `https://github.com/travelping/ergw/releases/tag/2.8.8`

Reply:
ah, ok